openapi-spec/swagger.yaml

```diff
@@ -2424,44 +2489,87 @@ components:
                 lvPath: string
                 nfsFileshareID: string
                 nfsFileshareName: string
-    FileShareSnapshotSpec:
+    FileShareSnapshotCreateSpec:
       description: Snapshot is a description of fileshare snapshot resource.
       allOf:
-        - $ref: '#/components/schemas/BaseModel'
-        - type: object
-          required:
-            - tenantId
-            - name
-            - status
-            - snapshotId
-            - fileshareId
-          properties:
-            fileshareId:
-              type: string
-            tenantId:
-              type: string
-              readOnly: true
-            userId:
-              type: string
-              readOnly: true
-            name:
-              type: string
-            description:
-              type: string
-            status:
-              type: string
-              readOnly: true
-            profileId:
+      - type: object
+        required:
+        - name
+        - fileshareId
+        properties:
+          fileshareId:
+            type: string
+          name:
+            minLength: 1
+            maxLength: 255
+            type: string
+          description:
+            type: string
+          profileId:
+            type: string
+          poolId:
+            type: string
+    FileShareSnapshotUpdateSpec:
+      description: Snapshot is a description of fileshare snapshot resource.
+      allOf:
+      - type: object
```

Comment:
remove allOf here

Reply:
Done

Reply:
this is not right to use `-` here. please remove it